### DISCUSSION TITLE
Give me the JSON response to determine what should happen

### DISCUSSION THREAD

**Discussion by Godrules500** [microsoft/teams-sdk#467] [Q&A] [answered]
**Please note: any submissions with insufficient reproducible information will be marked as 'Waiting for customer input' and may be closed is there is no response**

**Describe your user scenario**
I may just be missing this in the code samples, but in my scenario, I still want the AI to determine the action, but I also want it to return to me the JSON I told OpenAI to return. That way, I can then use those values to determine the correct sub actions. Our use case is that we won't have open ai respond to the user currently, so I need to tie into that layer and prevent it from speaking to the user as well.

**Comment by corinagum** [comment, marked as answer]
Please check out the [Light Bot sample](https://github.com/microsoft/teams-ai/blob/main/js/samples/04.ai.c.actionMapping.lightBot/src/prompts/chatGPT/skprompt.txt) for an example of AI returning the projected JSON.
